plugins/dash/packet-dash.c

```diff
@@ -622,6 +661,154 @@ static header_field_info hfi_msg_tx_extra_payload_size64 DASH_HFI_INIT =
+static header_field_info hfi_msg_tx_extra_proupregtx_inputshash DASH_HFI_INIT =
+  { "Inputs hash", "dash.tx.proupregtx.inputshash", FT_BYTES, BASE_NONE, NULL, 0x0, NULL, HFILL };
+static header_field_info hfi_msg_tx_extra_proupregtx_payloadsig_size8 DASH_HFI_INIT =
+  { "Signature Size", "dash.tx.proupregtx.payloadsigsize", FT_UINT8, BASE_DEC, NULL, 0x0, NULL, HFILL };
+static header_field_info hfi_msg_tx_extra_proupregtx_payloadsig_size16 DASH_HFI_INIT =
+  { "Signature Size", "dash.tx.proupregtx.payloadsigsize", FT_UINT16, BASE_DEC, NULL, 0x0, NULL, HFILL };
+static header_field_info hfi_msg_tx_extra_proupregtx_payloadsig_size32 DASH_HFI_INIT =
+  { "Signature Size", "dash.tx.proupregtx.payloadsigsize", FT_UINT32, BASE_DEC, NULL, 0x0, NULL, HFILL };
+static header_field_info hfi_msg_tx_extra_proupregtx_payloadsig_size64 DASH_HFI_INIT =
+  { "Signature Size", "dash.tx.proupregtx.payloadsigsize", FT_UINT64, BASE_DEC, NULL, 0x0, NULL, HFILL };
+static header_field_info hfi_msg_tx_extra_proupregtx_payloadsig DASH_HFI_INIT =
+  { "Signature", "dash.tx.proupregtx.payloadsig", FT_BYTES, BASE_NONE, NULL, 0x0, NULL, HFILL };
+
+/* tx.prouprevtx */
+static header_field_info hfi_msg_tx_extra_prouprevtx DASH_HFI_INIT =
+  { "ProUpRevTx Payload", "dash.tx.prouprevtx", FT_NONE, BASE_NONE, NULL, 0x0, NULL, HFILL };
+static header_field_info hfi_msg_tx_extra_prouprevtx_version DASH_HFI_INIT =
+  { "Version", "dash.tx.prouprevtx.version", FT_UINT16, BASE_DEC, NULL, 0x0, NULL, HFILL };
+static header_field_info hfi_msg_tx_extra_prouprevtx_protxhash DASH_HFI_INIT =
+  { "ProRegTx Hash", "dash.tx.prouprevtx.protxhash", FT_BYTES, BASE_NONE, NULL, 0x0, NULL, HFILL };
+static header_field_info hfi_msg_tx_extra_prouprevtx_reason DASH_HFI_INIT =
+  { "Version", "dash.tx.prouprevtx.reason", FT_UINT16, BASE_DEC, NULL, 0x0, NULL, HFILL };
+static header_field_info hfi_msg_tx_extra_prouprevtx_inputshash DASH_HFI_INIT =
+  { "ProRegTx Hash", "dash.tx.prouprevtx.inputshash", FT_BYTES, BASE_NONE, NULL, 0x0, NULL, HFILL };
+static header_field_info hfi_msg_tx_extra_prouprevtx_payloadsig DASH_HFI_INIT =
+  { "Signature", "dash.tx.prouprevtx.payloadsig", FT_BYTES, BASE_NONE, NULL, 0x0, NULL, HFILL };
+
+/* tx.cbtx */
+static header_field_info hfi_msg_tx_extra_cbtx DASH_HFI_INIT =
+  { "CbTx Payload", "dash.tx.cbtx", FT_NONE, BASE_NONE, NULL, 0x0, NULL, HFILL };
+static header_field_info hfi_msg_tx_extra_cbtx_version DASH_HFI_INIT =
+  { "Version", "dash.tx.cbtx.version", FT_UINT16, BASE_DEC, NULL, 0x0, NULL, HFILL };
+static header_field_info hfi_msg_tx_extra_cbtx_height DASH_HFI_INIT =
+  { "Block Height", "dash.tx.cbtx.height", FT_UINT32, BASE_DEC, NULL, 0x0, NULL, HFILL };
+static header_field_info hfi_msg_tx_extra_cbtx_merkle_mn DASH_HFI_INIT =
+  { "MN List merkle root", "dash.tx.cbtx.merklemn", FT_BYTES, BASE_NONE, NULL, 0x0, NULL, HFILL };
+static header_field_info hfi_msg_tx_extra_cbtx_merkle_quorum DASH_HFI_INIT =
```

Comment:
This is only present in CbTx [version 2](https://github.com/dashpay/dips/blob/master/dip-0004.md#coinbase-special-transaction). Doing an initial block download on testnet revealed this - blocks using CbTx v1 are flagged as malformed since they don't dissect properly.